### DISCUSSION TITLE
meeting-with-your-mentoring-partner/how-to-end-mentoring-relationship

### DISCUSSION THREAD

**Discussion by giscus** [OfferZen-Community/developer-mentoring#194] [Announcements]
# meeting-with-your-mentoring-partner/how-to-end-mentoring-relationship

How to Find a Mentor

https://www.developermentoring.guide/meeting-with-your-mentoring-partner/how-to-end-mentoring-relationship

**Comment by Legacyfgs** [comment]
I’m looking for mentor as I would love to find an internship as to enhance my skills
